packages/suir-component-mapper/src/common/form-field-grid.js

```diff
@@ -11,20 +11,21 @@ const useStyles = createUseStyles({
   }
 });
 
-const FormFieldGrid = ({ className, children, helperText }) => {
+const FormFieldGrid = ({ className, children, helperText, helperTextClassName }) => {
```

Comment:
what about do `HelperTextProps` instead of only className?

Reply:
Right